CONTRIBUTING.md

```diff
@@ -7,19 +7,11 @@ filing issues. Please read it carefully to help make the code review
 process go as smoothly as possible and maximize the likelihood of your
 contribution being merged.
 
-_Note:_  
-If you want to contribute new functionality, you may first consider if this 
-functionality belongs to the pydicom core, or is better suited for
-[contrib-pydicom](https://github.com/pydicom/contrib-pydicom). contrib-pydicom
-collects some convenient functionality that uses pydicom, but doesn't
-belong to the pydicom core. If you're not sure where your contribution belongs, 
-create an issue where you can discuss this before creating a pull request.
-
```

Comment:
I'm not sure if we should completely remove this. Some consideration if the change belongs to the `pydicom` core, or to another package using `pydicom` may remain. I have no clear opinion here, though.

Reply:
I think it's fine without it, worst case they'll create an issue or PR and we'll point them where to go.